Crashes immediately after loading screen finishes

Issue type: `problem`

### Description:

In both the v1.2.1, v1.3 x86, and v1.3 x64 "non-x86" versions, the game will crash immediately after the loading screen completes due to a RuntimeException. This occurs whenever the app.config generated by the game has a value greater than 0.0 for the "VOLUME=" setting within the file. Even the stock app.config file created by the game when cleanly installed / extracted, produces this behavior.

It should be noted that the bug only occurred on Windows systems, being personally observed on both Windows 7 Home Premium 64-bit Edition, and Windows 10, while using Oracle's Java 8. When tested on Debian Linux 8 (Jessie) 64-bit, with OpenJDK8 (*which includes OpenJRE8*) the game ran without crashing at all, regardless of the value assigned to "VOLUME=".

That being said, I'm led to believe that this could be a compatibility issue with Oracle's Java, or Windows itself. I can't find a Windows binary of OpenJDK8 to confirm or deny this theory though. Please advise.

Edited to fix typos.

-------

#### Examples:

Raw text from error_log_client3464147.log:
````
java.lang.RuntimeException: java.lang.NullPointerException
	at net.fe.SoundTrack.loadAudioNames(SoundTrack.java:106)
	at net.fe.SoundTrack.loop(SoundTrack.java:46)
	at net.fe.FEMultiplayer.setCurrentStage(FEMultiplayer.java:392)
	at net.fe.FEMultiplayer.init(FEMultiplayer.java:146)
	at net.fe.FEMultiplayer.main(FEMultiplayer.java:96)
Caused by: java.lang.NullPointerException
	at net.fe.SoundTrack.loadAudioNames(SoundTrack.java:95)
	... 4 more
````
##### Windows CMD Output: (*Crashes upon transition to title screen where music is suppose to play*)

    C:\Users\Drew\Desktop\Fire Emblem Multiplayer>java -jar Fire_Emblem_Multiplayer_V2.jar
    Wed Apr 20 14:47:55 EDT 2016 INFO:Use Java PNG Loader = true
    default_med(bitmap font) loaded
    stat_numbers(bitmap font) loaded
    msg_text(bitmap font) loaded
    1.20 NVIDIA via Cg compiler
    20
    Wed Apr 20 14:47:56 EDT 2016 INFO:Initialising sounds..
    Wed Apr 20 14:47:57 EDT 2016 INFO:- Sound works
    Wed Apr 20 14:47:57 EDT 2016 INFO:- 64 OpenAL source available
    Wed Apr 20 14:47:57 EDT 2016 INFO:- Sounds source generated
    default_med(bitmap font) loaded
    stat_numbers(bitmap font) loaded
    msg_text(bitmap font) loaded
    Loaded res/gui/rescue.png
    Loaded res/gui/runes.png
    Loaded res/gui/connect_button.png
    Loaded res/gui/team_builder_button.png
    Loaded res/gui/title.png
    Loaded res/gui/title_two.png
    Loaded res/gui/title_slide.png
    Exception occurred, writing to logs...
    java.lang.RuntimeException: java.lang.NullPointerException
            at net.fe.SoundTrack.loadAudioNames(SoundTrack.java:106)
            at net.fe.SoundTrack.loop(SoundTrack.java:46)
            at net.fe.FEMultiplayer.setCurrentStage(FEMultiplayer.java:392)
            at net.fe.FEMultiplayer.init(FEMultiplayer.java:146)
            at net.fe.FEMultiplayer.main(FEMultiplayer.java:96)
    Caused by: java.lang.NullPointerException
            at net.fe.SoundTrack.loadAudioNames(SoundTrack.java:95)
            ... 4 more
    AL lib: (EE) alc_cleanup: 1 device not closed
    
    C:\Users\Drew\Desktop\Fire Emblem Multiplayer>

These results remained consistent with both the JRE and JDK.

For comparison's sake, when ran successfully through my Linux terminal, I get the following.

##### Linux Terminal Output: (*Successfully reaches title screen*)

    drew@Mint-Drop:~$ cd .Fire_Emblem_Multiplayer_V2/
    drew@Mint-Drop:~/.Fire_Emblem_Multiplayer_V2$ java -jar Fire_Emblem_Multiplayer_V2.jar
    Wed Apr 20 15:20:21 EDT 2016 INFO:Use Java PNG Loader = true
    default_med(bitmap font) loaded
    stat_numbers(bitmap font) loaded
    msg_text(bitmap font) loaded
    Wed Apr 20 15:20:21 EDT 2016 INFO:Initialising sounds..
    Wed Apr 20 15:20:21 EDT 2016 INFO:- Sound works
    Wed Apr 20 15:20:21 EDT 2016 INFO:- 64 OpenAL source available
    Wed Apr 20 15:20:21 EDT 2016 INFO:- Sounds source generated
    default_med(bitmap font) loaded
    stat_numbers(bitmap font) loaded
    msg_text(bitmap font) loaded
    Loaded res/gui/rescue.png
    Loaded res/gui/runes.png
    Loaded res/gui/connect_button.png
    Loaded res/gui/team_builder_button.png
    Loaded res/gui/title.png
    Loaded res/gui/title_two.png
    Loaded res/gui/title_slide.png
    1.20 NVIDIA via Cg compiler
    20

Upon successfully reaching the title screen in Linux ONLY, complete with music, I closed it as this was just to show that Linux can reach the title screen with the music enabled just fine with the default settings. I noticed some messages appear in a different order when under Linux, not sure if that's relevant but figured I'd show you anyways.

--- a/src/net/fe/SoundTrack.java
+++ b/src/net/fe/SoundTrack.java
@@ -72,7 +72,7 @@ public class SoundTrack {
 		try{
 			songs = new HashMap<String, ArrayList<String>>();
 			final String musPath = "res/music";
-			final File jarFile = new File(SoundTrack.class.getProtectionDomain().getCodeSource().getLocation().getPath());
+			final File jarFile = new File(SoundTrack.class.getProtectionDomain().getCodeSource().getLocation().toURI());
 			if(jarFile.isFile()) {  // Run with JAR file
 			    final JarFile jar = new JarFile(jarFile);
 			    final Enumeration<JarEntry> entries = jar.entries(); //gives ALL entries in jar
@@ -89,7 +89,7 @@ public class SoundTrack {
 			    }
 			    jar.close();
 			    
-			} else { // Run with IDE
+			} else if (new File(musPath).isDirectory()) { // Run with IDE
 				File folder = new File(musPath);
 				File[] listOfFiles = folder.listFiles();
 				for(File s: listOfFiles){

